build/Microsoft.DotNet.Cli.Monikers.props

```diff
@@ -3,8 +3,8 @@
   <PropertyGroup>
     <SdkBrandName>Microsoft .NET Core 1.0.3 - SDK RC 4</SdkBrandName>
```

Comment:
@blackdwarf @leecow Do we need to update these brand names at all, now that we ship both 1.0.3 and 1.1.0 runtimes in the CLI?